openerp/tools/mail.py

```diff
@@ -292,6 +293,11 @@ def _truncate_node(node, position, simplify_whitespaces=True):
 
     if expand_options is None:
         expand_options = {}
+    whitelist_classes_local = []
+    if expand_options.get('oe_expand_container_class'):
+        whitelist_classes_local.append(expand_options.get('oe_expand_container_class'))
+    if expand_options.get('oe_expand_a_class'):
+        whitelist_classes_local.append(expand_options.get('oe_expand_a_class'))
```

Comment:
Please use a `set` instead of a list when used only for testing inclusion - and a single set for both local and global classes would be good enough. You could clone the set at the beginning of the function and add the "local" classes in it according to the options.
This probably does not give a measurable speed boost, but it's a best practice everywhere, and doesn't make the code more complex.